community.md

```diff
@@ -35,6 +36,7 @@ Below is a list of projects and resources from the Blazor community (links in ea
 * [ChartJs.Blazor](https://github.com/mariusmuntean/ChartJs.Blazor) &ndash; Blazor Component that wraps ChartJS.
 * [Blazor Extensions Home](https://github.com/BlazorExtensions/Home) &ndash; Home for Blazor Extensions.
 * [Logging](https://github.com/BlazorExtensions/Logging) &ndash; Microsoft Extension Logging implementation for Blazor.
+* [MatBlazor](https://github.com/BlazorComponents/MatBlazor) ([Demo](https://blazorcomponents.github.io/MatBlazor/)) ([nuget.org](https://www.nuget.org/packages/MatBlazor)) &ndash; Google Material components for web in Blazor.
```

Comment:
We don't typically link to the NuGet package here. We've been linking primarily to the GitHub repo, and then optionally to a live demo site.

Reply:
I have removed nuget packages!

Reply:
nuget packages removed!